assets/js/modules/ads/components/notifications/PAXSetupSuccessSubtleNotification.js

```diff
@@ -61,44 +60,25 @@ export default function PAXSetupSuccessSubtleNotification() {
 	}
 
 	return (
-		<Grid>
-			<Row>
-				<Cell
-					alignMiddle
-					size={ 12 }
-					className="googlesitekit-subtle-notification"
+		<SubtleNotification
+			title={ __(
+				'Your Ads campaign was successfully set up!',
+				'google-site-kit'
+			) }
+			description={ __(
+				'Track your conversions, measure your campaign results and make the most of your ad spend',
+				'google-site-kit'
+			) }
+			onDismiss={ onDismiss }
+			additionalCTA={
+				<Button
+					onClick={ ( event ) => {
+						scrollToWidget( event );
+					} }
```

Comment:
```suggestion
					onClick={ scrollToWidget }
```

Reply:
Thanks @eugene-manuilov sorted in latest commit.